randovania/games/dread/json_data/Artaria.txt

```diff
@@ -593,7 +593,12 @@ Extra - asset_id: collision_camera_005
   > Door to EMMI Zone Entrance Hallway
       Can Slide
   > Next to Pickup
-      Grapple Beam or Spider Magnet or Space Jump or Simple IBJ
+      Any of the following:
+          Spider Magnet or Space Jump or Simple IBJ
+          Grapple Beam and Grapple Movement (Beginner)
+          Walljump (Beginner) and Use Spin Boost
+          Flash Shift and Walljump (Intermediate)
+          Morph Ball and Single-wall Wall Jump (Intermediate)
```

Comment:
Couldn´t we add Pseudo Wave (Beginner) + Legde jump (Beginner)?

Reply:
Unsure what a pseudo or a ledge jump (which is not even a trick) would do here. You're climbing the walls up, not shooting anything.

Reply:
Coming from early cloak and jumping up there is what I meant with this